tests/user_commands/management/commands/optparse_cmd.py

```diff
@@ -16,4 +16,5 @@ def handle(self, *args, **options):
         options["example"]
         # BaseCommand default option is available
         options['verbosity']
+        assert isinstance(options['verbosity'], int), "verbosity option is not int, but %s" % type(options['verbosity'])
```

Comment:
please use `self.assertIsInstance(options['verbosity'])` instead of plain assert statements.